packages/sample-app/sample_app/mcp_dev_assistant_server.py

```
@@ -0,0 +1,387 @@
+    Args:
+        command: Shell command to execute
+        working_directory: Directory to run the command in (default: current)
+        timeout: Command timeout in seconds (default: 30)
+    
+    Returns:
+        Process execution result with stdout, stderr, and exit code
+    """
+    try:
+        start_time = datetime.now()
+        
+        result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)